services/src/main/webapp/WEB-INF/web.xml

```diff
@@ -1,138 +1,138 @@
+    </filter>
+    <filter-mapping>
+        <filter-name>storeAccessLevelFilter</filter-name>
+        <url-pattern>/api/status/*</url-pattern>
+        <url-pattern>/api/edit/*</url-pattern>
+    </filter-mapping>
+    
+    <filter>
+        <filter-name>SwordHeaderRewriteFilter</filter-name>
+        <filter-class>edu.unc.lib.dl.cdr.sword.server.filters.SwordHeaderRewriteFilter</filter-class>
+    </filter>
+    <filter-mapping>
+        <filter-name>SwordHeaderRewriteFilter</filter-name>
+        <url-pattern>/sword/*</url-pattern>
+    </filter-mapping>
+    
+    <filter>
+        <filter-name>depositorAccessControlFilter</filter-name>
+        <filter-class>org.springframework.web.filter.DelegatingFilterProxy</filter-class>
+    </filter>
+    <filter-mapping>
+        <filter-name>depositorAccessControlFilter</filter-name>
+        <url-pattern>/sword/*</url-pattern>
+    </filter-mapping>
 
-	<!-- SWORD basic authentication configuration -->
-	<security-constraint>
-		<web-resource-collection>
-			<web-resource-name>SWORD Server</web-resource-name>
-			<url-pattern>/sword/*</url-pattern>
-		</web-resource-collection>
-		<auth-constraint>
-			<role-name>sword-depositor</role-name>
-		</auth-constraint>
-	</security-constraint>
-	
-	<security-constraint>
-		<web-resource-collection>
-			<web-resource-name>Forwarding API</web-resource-name>
-			<url-pattern>/api-s/*</url-pattern>
-		</web-resource-collection>
-		<auth-constraint>
-			<role-name>group-forwarding</role-name>
-		</auth-constraint>
-	</security-constraint>
-	
-	<login-config>
-		<auth-method>BASIC</auth-method>
-		<realm-name>ServicesRealm</realm-name>
-	</login-config>
-	
-	<security-role>
-		<role-name>sword-depositor</role-name>
-	</security-role>
-	
-	<security-role>
-		<role-name>group-forwarding</role-name>
-	</security-role>
+    <!-- SWORD basic authentication configuration -->
+    <security-constraint>
+        <web-resource-collection>
+            <web-resource-name>SWORD Server</web-resource-name>
+            <url-pattern>/sword/*</url-pattern>
+        </web-resource-collection>
+        <auth-constraint>
+            <role-name>sword-depositor</role-name>
+        </auth-constraint>
+    </security-constraint>
+    
+    <security-constraint>
+        <web-resource-collection>
+            <web-resource-name>Forwarding API</web-resource-name>
+            <url-pattern>/api-s/*</url-pattern>
+        </web-resource-collection>
+        <auth-constraint>
+            <role-name>group-forwarding</role-name>
+        </auth-constraint>
+    </security-constraint>
+    
+    <login-config>
+        <auth-method>BASIC</auth-method>
+        <realm-name>ServicesRealm</realm-name>
+    </login-config>
+    
+    <security-role>
+        <role-name>sword-depositor</role-name>
+    </security-role>
+    
+    <security-role>
+        <role-name>group-forwarding</role-name>
+    </security-role>
```

Comment:
I guess this PR is changing the indent characters in xml files in the impacted modules?  Well, lets go through and do that to all the other context files in another PR so that we can set eclipse to indent consistently after this.